docs/config.md

```diff
@@ -24,3 +24,12 @@ The following environment variables are available to configure the NetObserv eBF
   `error`, `fatal`, `panic`.
 * `BUFFERS_LENGTH` (default: `50`). Length of the internal communication channels between the different
   processing stages. Most probably you won't need to change this value.
+* `LISTEN_INTERFACES` (default: `watch`). Mechanism used by the agent to listen for added or removed
+  network interfaces. Accepted values are:
+  - `watch`: interfaces are traced immediately after they are created. This is
+    the recommended setting for most configurations.
+  - `poll`: recommended mostly as a fallback mechanism if `watch` misbehaves. It periodically 
+    queries the current network interfaces. The poll frequency is specified by the
+    `LISTEN_POLL_PERIOD` variable.
+* `LISTEN_POLL_PERIOD` (default: `10s`). When `LISTEN_INTERFACES` value is `poll`, this duration
+  string specifies the frequency in which the current network interfaces are polled.
```

Comment:
We should expose that in the operator config

Reply:
The operator config now allows setting un an `env` section where you can set the Agent environment variables. On a first glance I decided to keep them just as env vars, hidden from the operator spec, to simplify the NOO documentation and hide from the common user some performance-tweaking arguments that are aimed for some very specific configurations (I'm thinking in customer support).